### DISCUSSION TITLE
Quorum mechanism support

### DISCUSSION THREAD

**Discussion by Vovchikus** [php-amqplib/php-amqplib#1098] [Q&A] [answered]
Does this library supports RMQ version 3.12 and Quorum mechanism of queues (also with lazy queues and retries during replication)?

**Comment by lukebakken** [comment, marked as answer]
> Does this library supports RMQ version 3.12

Yes

> and Quorum mechanism of queues

Yes, you must declare your queues using the `x-queue-type: quorum` argument - https://www.rabbitmq.com/quorum-queues.html

> also with lazy queues and retries during replication

I'm not quite sure what you're asking here.
